Community/gitlab_import/gitlab_import_config.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 BlueCat Networks (USA) Inc. and its affiliates
```

Comment:
Not sure if you want all of these values hardcoded in a config file, it might be better if the user can alter them in the UI